openwebstart/src/main/java/com/openwebstart/os/mac/MacEntryFactory.java

```diff
@@ -25,29 +25,38 @@
 public class MacEntryFactory implements MenuAndDesktopEntriesFactory {
 
     @Override
-    public boolean supportsDesktopEntry() {
-        return false;
+    public boolean supportsDesktopEntry() 
+    {
+        return true;
```

Comment:
Let's switch this back to false. By doing so we can test the behaviour internally and activate the support with a Pull Request in near future :)